package.json

```diff
@@ -13,7 +13,9 @@
     "watch": "tsc -w",
     "dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"TypeScript,App\" -c \"yellow.bold,cyan.bold\" \"npm:watch\" \"npm:dev:start\"",
     "dev:start": "npm run build && fastify start --ignore-watch=.ts$ -w -l info -P dist/app.js",
-    "test": "npm run db:seed && tap --jobs=1 test/**/*",
+    "test": "npm run db:seed && npm run test:run",
+    "test:coverage": "npm run db:seed && c8 npm run test:run",
```

Comment:
I would prefer to run coverage in `test` script, I want contributor to be aware that we impose high code coverage before they open a PR.